ee/codegen_integration/conftest.py

```diff
@@ -37,6 +37,7 @@ def _get_fixture_paths(root: str) -> Tuple[str, str]:
             "simple_code_execution_node",
             # TODO: Remove once graph is fleshed out. Current the workflow is not pointing to any port branch
             "simple_conditional_node",
+            "simple_templating_node",
```

Comment:
@peicodes I'll need your help removing this from the exclusion list after we've fixed templating node serialization issues.